### DISCUSSION TITLE
Support for native C++

### DISCUSSION THREAD

**Discussion by roshannzth** [CommunityToolkit/Windows#210] [Q&A] [answered]
Any plans to support native C++, as existing win ui 3 supports lot of their controls.

I tried installing the nuget package, but got the below error
`Could not install package 'CommunityToolkit.Labs.WinUI.SegmentedControl 0.1.230830'. You are trying to install this package into a project that targets 'native,Version=v0.0', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.`

**Comment by michael-hawker** [comment, marked as answer]
@roshannzth no, we don't plan to add support for C++. The Toolkit is a .NET Foundation project and focused on C# development only. Controls that get widely used have ended up in the platform WinUI library in the past, so if this is something you think many folks need, it'd be best to open a request with the platform to gather support there. Then we can work with them to transfer the component to the platform SDK in the future if they're interested.
